encoder/vaapiencoder_h264.cpp

```diff
@@ -1319,9 +1321,10 @@ bool VaapiEncoderH264::addSliceHeaders (const PicturePtr& picture) const
         sliceParam->slice_qp_delta = initQP() - minQP();
         if (sliceParam->slice_qp_delta > 4)
             sliceParam->slice_qp_delta = 4;
-        sliceParam->slice_alpha_c0_offset_div2 = 2;
-        sliceParam->slice_beta_offset_div2 = 2;
 
+        sliceParam->disable_deblocking_filter_idc = !m_videoParamAVC.enableDeblockFilter;
+        sliceParam->slice_alpha_c0_offset_div2 = m_videoParamAVC.deblockAlphaOffset / 2;
```

Comment:
I've no comment to use deblockAlphaOffset or sliceAlphaC0OffsetDiv2. But we would better comment how this name maps h264 spec. For example, if sliceAlphaC0OffsetDiv2 used, comments "same as slice_alpha_c0_offset_div2 defined in h264 spec 7.4.3"